maintainers/information-flow.md

```diff
@@ -0,0 +1,53 @@
+# Information flow in the Nix ecosystem
+
+The diagram shows a simplified model of the information flow in the Nix ecosystem.
+Its purpose is to make dependencies of the documentation types more explicit.
+Note that it does not show issues and pull requests on GitHub, which host a wealth of ephemeral information.
+
+Nodes are places or resources, edges denote an "informed by" relation, and edge labels are units of information.
+Read the example as "A is informed by X and placed in Y":
+ 
+```mermaid
+flowchart LR
+  X --> |A| Y
+```
+
+Information resources – communication platforms – are distinguished by mode of interaction:
+
+*Streams* can be appended to:
+
+- The amount of information in them grows without bounds.
+- Only recent items are relevant.
```

Comment:
Minor thing: is this true for RFCs as well? They're categorised as a stream.
I think RFCs should be considered artifacts. I'm interpreting RFCs as each individual RFC instead of the "RFC repository". In that sense, they're more like artifacts.

Reply:
I bounced back and forth. I'm inclined to consider RFCs throwaway, because ideally (and this is just parroting comments from other people) the knowledge gets merged into living code and documentation. That they aren't in practice seems rather like a slight dysfunction of processes.